src/test/java/dev/openfeature/sdk/TrackingSpecTest.java

```diff
@@ -0,0 +1,187 @@
+package dev.openfeature.sdk;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.Maps;
+import dev.openfeature.sdk.fixtures.ProviderFixture;
+import dev.openfeature.sdk.testutils.FeatureProviderTestUtils;
+import lombok.SneakyThrows;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThatCode;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.ArgumentMatchers.notNull;
+import static org.mockito.Mockito.any;
+import static org.mockito.Mockito.argThat;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.verify;
+
+class TrackingSpecTest {
+
+    private OpenFeatureAPI api;
+    private Client client;
+
+    @BeforeEach
+    void getApiInstance() {
+        api = OpenFeatureAPI.getInstance();
+        client = api.getClient();
+    }
+
+
+    @Specification(number = "6.1.1.1", text = "The `client` MUST define a function for tracking the occurrence of " +
```

Comment:
```suggestion
    @Specification(number = "6.1.1", text = "The `client` MUST define a function for tracking the occurrence of " +
```
I think we do not need the last digit

Reply:
Technically this is correct: https://openfeature.dev/specification/sections/tracking/#conditional-requirement-6111